compiler/AST/view.cpp

```diff
@@ -26,6 +26,7 @@
 #include "AstDump.h"
```

Comment:
Why did view.cpp need any of these changes?

If they are not needed, please revert view.cpp,
to reduce the diffs.